### PR TITLE
Added slack controller to v2 API

### PR DESCRIPTION
**Pull request by rshbhgrg**
refs #10060

- Added slack controller to v2 admin API
- Added new API test for slack API controller

### CODE REVIEW

**Comment by gargol** [comment]
small and clean :) :ship:

File: core/server/api/v2/slack.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const common = require('../../lib/common');
+
+module.exports = {
+    docName: 'slack',
+    sendTest: {
+        permissions: false,
+        query() {
+            common.events.emit('slack.test');
+            return Promise.resolve();
```
Comment (gargol) [inline core/server/api/v2/slack.js]:
```suggestion
```

Had a second thought on this part. We don't have to return anything from the query function in this situation, would remove unneeded `return Promise.resolve();`

File: core/server/api/v2/utils/serializers/output/slack.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:slack');
```
Comment (gargol) [inline core/server/api/v2/utils/serializers/output/slack.js]: seems like this whole serializer is also kind of redundant, would not add the code that is not needed :)  we don't expect anything to be returned for this endpoint anyways

File: core/server/api/v2/utils/serializers/output/index.js [hunk 3]
```diff
@@ -57,5 +57,9 @@ module.exports = {
 
     get preview() {
         return require('./preview');
+    },
+
+    get slack() {
```
[the remark this answers is not in the document]
Comment (rishabhgrg) [reply core/server/api/v2/utils/serializers/output/index.js]: @gargol Updated with all suggestions 😄

**Pull request merged**